src/aura/BGE_BatchGiftEntryTab/BGE_BatchGiftEntryTab.css

```diff
@@ -1,3 +1,11 @@
 .THIS .slds-scrollable_y {
     min-height: 200px
+}
+
+#retirementPrompt.THIS {
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    z-index: 1000;
+    max-width: 450px;
```

Comment:
I haven't deployed this and tested in the UI yet - but what is the behavior if the viewport is shrunk down significantly?  Does the prompt stay in the same place as the window size changes?

Reply:
Good question. The prompt stays fixed to the bottom right as the viewport initially shrinks -- and then as the prompt nears the size of the viewport, it shifts to fill up the space.

**Full Viewport**

<img width="1440" alt="Screen Shot 2021-03-09 at 9 40 10 AM" src="https://user-images.githubusercontent.com/23663916/110513951-a1fec480-80bb-11eb-95ea-be3c93e78a5e.png">

**Minimized Viewport**

<img width="497" alt="Screen Shot 2021-03-09 at 9 40 17 AM" src="https://user-images.githubusercontent.com/23663916/110514006-b04ce080-80bb-11eb-8e17-5dde1a11781a.png">

Please let me know if you think any adjustments need to made here. Completely open to exploring other options if there is a better path forward.

Reply:
I think this looks good - thanks for verifying.  Lets double check with Lauren - maybe in our review mtg on Thursday - and have her confirm that this looks good also.  Thanks!